src/nvim/ex_cmds.c

```diff
@@ -4833,7 +4833,7 @@ int find_help_tags(char_u *arg, int *num_matches, char_u ***matches, int keep_la
 {
   char_u      *s, *d;
   int i;
-  static char *(mtable[]) = {"*", "g*", "[*", "]*", ":*",
+  static char *(mtable[]) = {"*", "g*", "[*", "]*",
                              "/*", "/\\*", "\"*", "**",
                              "cpo-*", "/\\(\\)", "/\\%(\\)",
```

Comment:
The `"cpo-*",` field should be removed now that the help option is gone. The for loop that parses this below has a helpful comment:
```
 /*
    * Recognize a few exceptions to the rule.	Some strings that contain '*'
    * with "star".  Otherwise '*' is recognized as a wildcard.
    */
```